Add 4.7.1 developer pack to the rider setup (#853)
spatialos/gdk-for-unity: docs/setup-and-installing.md

**Using Rider?**

> Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [.NET Framework 4.7.1 Developer Pack](https://dotnet.microsoft.com/download/visual-studio-sdks).

Reviewer: ```suggestion
If you prefer to use [Rider](https://www.jetbrains.com/rider/), install the [.NET Framework 4.7.1 Developer Pack](https://dotnet.microsoft.com/download/visual-studio-sdks).
```

> You may also install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.

Reviewer: ```suggestion
Additionally, install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.
```
